examples/communication/ip/ReceiveIPDataSample/ReadMe.txt

```diff
@@ -0,0 +1,89 @@
+  Introduction
+  ------------
+  This sample Java application shows how IP data messages are received from 
+  another XBee device connected to the Internet.
+
+  The application prints the received IP data to the standard output in ASCII 
+  and hexadecimal formats after the sender IP address.
+
+  NOTE: This example uses the Wi-Fi device (WiFiDevice) class, but it can be 
+        applied to other Internet capable XBee device classes such as 
+        CellularDevice.
+
+
+  Files
+  -----
+    * com.digi.xbee.api.receiveipdata.MainApp.java:
+      Main application class. It instantiates a Wi-Fi device and establishes a 
+      serial connection with it.
+
+    * com.digi.xbee.api.receiveipdata.MyIPDataReceiveListener.java:
+      Class that handles the received IP data messages.
+
+
+  Requirements
+  ------------
+  To run this example you will need:
+
+    * At least two Internet capable XBee radios (for example Wi-Fi radios) in 
```

Comment:
The example is prepared for Wi-Fi modules, so I would change this line by
```
At least two XBee Wi-Fi radios
```
I have checked the ReadMe of the ConnectToEchoServerSample and I think we should change it there as well. In other examples we were using the `XBeeDevice` generic class and it didn't matter the kind of module we were connecting. But in these examples we are using final protocol classes such as `WiFiDevice` and `CellularDevice`. If the user connects a module of a different protocol than the class one the example will fail.